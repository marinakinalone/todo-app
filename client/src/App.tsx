import React from 'react';
import './css/main.css';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import { Summary, TodoList } from './components/Index';

function App() {
  return (
    <Router>
    <div className="App">
        <Routes>
          <Route path="/" element={<Summary />} />
          <Route path="/:id" element={<TodoList />} />
        </Routes>
      </div>
    </Router>
  );
}

export default App;
