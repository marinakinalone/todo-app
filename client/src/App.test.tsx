import App from './App';
import renderer from 'react-test-renderer';

describe('renders all the components', () => {
  test('snapshot test for App', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  })
});