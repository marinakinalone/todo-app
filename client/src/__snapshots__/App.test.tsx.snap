// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the components snapshot test for App 1`] = `
<div
  className="App"
>
  <main
    className="page__todos"
  >
    <header
      className="header"
    >
      <h1
        className="header__title"
      >
        my to-do lists
      </h1>
    </header>
    <p>
      loading todo lists...
    </p>
  </main>
</div>
`;
